openapi-cli/src/test/resources/generators/client/auth/scenarios/oauth2/oauth2_authorization_code_without_tokenurl.yaml

```diff
@@ -0,0 +1,118 @@
+          style: simple
+          explode: false
+          schema:
+            type: string
+        - name: version
+          in: path
+          description: version
+          required: true
+          style: simple
+          explode: false
+          schema:
+            type: string
+            default: v48.0
+            enum:
+              - v47.0
+              - v48.0
+      responses:
+        "200":
+          description: "Status Okay"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Account_inner'
+        "400":
+          description: The request couldn’t be understood, usually because the JSON or XML body contains an error.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/MessagewithErrorCode'
+        "401":
+          description: The session ID or OAuth token used has expired or is invalid. The response body contains the message and errorCode.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/MessagewithErrorCode'
+      security:
+        - oAuth2AuthCode: []
+components:
+  schemas:
+    attributes:
+      required:
+        - type
+        - url
+      type: object
+      properties:
+        type:
+          type: string
+          description: the type of object.
+          example: Account
+        url:
+          type: string
+          description: The relative path of the object.
+          example: /services/data/v46.0/sobjects/Account/0010E00000Up3QGQAZ
+    MessagewithErrorCode:
+      properties:
+        code:
+          type: string
+          description: Code
+    Account_inner:
+      required:
+        - Attributes
+        - Id
+        - Name
+      type: object
+      properties:
+        Attributes:
+          $ref: '#/components/schemas/attributes'
+        Id:
+          type: string
+          description: The Salesforce ID of the account
+          example: 0010E00000Up3QGQAZ
+        Name:
+          type: string
+          description: 'Name of the account. Maximum size is 255 characters. If the account has a record type of Person Account: This value is the concatenation of the FirstName, MiddleName, LastName, and Suffix of the associated person contact. You can''t modify this value.'
+          example: Klaas Vaak
+  responses: {}
+  parameters: {}
+  examples: {}
+  requestBodies: {}
+  headers: {}
+  securitySchemes:
+    oAuth2AuthCode:
+      type: oauth2
+      description: For more information, see https://api.slack.com/docs/oauth
+      flows:
+        authorizationCode:
+          authorizationUrl: https://login.salesforce.com/services/oauth2/autherize
+          tokenUrl: ""
+          scopes: {}
```

Comment:
Shall we add a new line there and other relevant places?

Reply:
Added new lines at bottom